src/test/java/wooteco/subway/station/StationDaoTest.java

```diff
@@ -15,8 +15,9 @@
 @SpringBootTest
 class StationDaoTest {
 
-    private static final String stationName1 = "잠실역";
-    private static final String stationName2 = "서울역";
+    private static final String STATIONNAME1 = "잠실역";
```

Comment:
```suggestion
    private static final String STATION_NAME1 = "잠실역";
```

또는 테스트 코드니까 그냥 `잠실역` 한글 변수명 사용해도 되지 않을까요?

Reply:
주어진 테스트코드들을 보면 한글명으로 다 되어있었는데 이걸로 해도 되는지 아리송 했어요! 다음번 테스트코드 부터는 한글명 적극 사용하겠습니다~!